src/software/ai/passing/cost_function_test.cpp

```diff
@@ -147,7 +147,8 @@ TEST_F(PassingEvaluationTest, ratePass_speed_test)
 
     std::chrono::duration<double> duration = end_time - start_time;
 
-    std::chrono::duration<double> avg = duration / (double)num_passes_to_gen;
+    std::chrono::duration<double> avg;
+    avg = duration / (double)num_passes_to_gen;
```

Comment:
Nit:

Do we care about using a static_cast here?

Reply:
Yes we should. Our style guide prefers `static_cast` in all possible cases.